features/http_cache/tags.feature

```diff
@@ -0,0 +1,64 @@
+Feature: Cache invalidation trough HTTP Cache tags
+  In order to have a fast API
+  As an API software developer
+  I need to store API responses in a cache
+
+  @createSchema
+  Scenario: Create some embedded resources
+    When I add "Content-Type" header equal to "application/ld+json"
+    And I send a "POST" request to "/relation_embedders" with body:
+    """
+    {
+      "anotherRelated": {
+        "name": "Related",
+        "thirdLevel": {}
+      }
+    }
+    """
+    Then the response status code should be 201
+    And the header "Cache-Tags" should not exist
+    And "/relation_embedders,/related_dummies,/third_levels,/relation_embedders/1,/related_dummies/1,/third_levels/1" IRIs should be purged
```

Comment:
I don't understand in you code how you make the link between `relation_embedders` and `third_levels`? Who flaged it?